componentLibraries/defaultLibrary/Electric/ElectricACmachine.hpp

```diff
@@ -167,7 +167,7 @@ class ElectricACmachine : public ComponentQ
         mpPel2=addPowerPort("Pel2","NodeElectric");
         mpPmr1=addPowerPort("Pmr1","NodeMechanicRotational");
         //Add inputVariables to the component
-            addInputVariable("freq","AC frequency","1/s",50,&mpfreq);
+            addInputVariable("freq","AC frequency","Hz",50,&mpfreq);
```

Comment:
Could write Frequency instead of unit, also should be fixed in source modelica file by @petkr14  (also in other files)
When Quantity is known, it should be used instead of "base SI unit" (Quantities are fairly "new") so lots of old code needs to be updated.

Reply:
That is correct, but I believe Peter meant to write "Mathematica" :) We have a Modelica generator as well, but I don't think any component in the default library is currently generated from there.

Anyway, both "Frequency" and "Hz" are better than "1/s", since it is unclear if "1/s" is a frequency or an angular frequency.